web/packages/extension/manifest_firefox.json5

```diff
@@ -48,7 +48,8 @@
         "open_in_tab": true,
     },
     "permissions": [
-        "<all_urls>", // // To allow script injecting + the internal player to bypass CORS
+        "<all_urls>", // // To allow script injecting + the internal player to bypass CORS + SWF takeover
+        "declarativeNetRequest",
```

Comment:
Completely unused on Firefox because Firefox does not yet support the `responseHeaders` `RuleCondition`, but adding it is not a problem because Firefox will throw within the `try` in the background script.